Gemfile.lock

```diff
@@ -1315,8 +1315,8 @@ GEM
     method_source (0.9.2)
     mime-types (3.3.1)
       mime-types-data (~> 3.2015)
-    mime-types-data (3.2021.0212)
-    mimemagic (0.3.5)
+    mime-types-data (3.2021.0225)
+    mimemagic (0.3.6)
```

Comment:
mimemagic (0.3.5 ) gem version was deprecated today. So updated to latest version.  
Breaking all CI installs of Rails - Fixed